recipes/usockets/all/conanfile.py

```diff
@@ -1,35 +1,47 @@
-import os
+from conan import ConanFile
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy, rmdir, chdir
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.errors import ConanInvalidConfiguration
+from conans import MSBuild, AutoToolsBuildEnvironment
 
-from conans import ConanFile, tools, MSBuild, AutoToolsBuildEnvironment
-from conans.errors import ConanInvalidConfiguration
+import os
 
+required_conan_version = ">=1.52.0"
 
 class UsocketsConan(ConanFile):
     name = "usockets"
-    url = "https://github.com/conan-io/conan-center-index"
     description = "Miniscule cross-platform eventing, networking & crypto for async applications"
     license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/uNetworking/uSockets"
-    topics = ("conan", "socket", "network", "web")
+    topics = ("socket", "network", "web")
     settings = "os", "arch", "compiler", "build_type"
-    options = {"fPIC": [True, False],
-               "with_ssl": [False, "openssl", "wolfssl"],
-               "with_libuv": [True, False, "deprecated"],
-               "eventloop": ["syscall", "libuv", "gcd", "boost"]}
-    default_options = {"fPIC": True,
-                       "with_ssl": False,
-                       "with_libuv": "deprecated",
-                       "eventloop": "syscall"}
+    options = {
+        "fPIC": [True, False],
+        "with_ssl": [False, "openssl", "wolfssl"],
+        "with_libuv": [True, False, "deprecated"],
```

Comment:
I think it's a good time to drop it it's been a few months

Reply:
Thank you!
I removed with_libuv option.